bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/ScanAndCompareGarbageCollector.java

```diff
@@ -101,14 +101,16 @@ public ScanAndCompareGarbageCollector(LedgerManager ledgerManager, CompactableLe
         if (gcOverReplicatedLedgerIntervalMillis > 0) {
             this.enableGcOverReplicatedLedger = true;
         }
+        this.maxConcurrentRequests = conf.getGcOverreplicatedLedgerMaxConcurrentRequests();
         this.zkServers = ZKMetadataDriverBase.resolveZkServers(conf);
         this.zkLedgersRootPath = ZKMetadataDriverBase.resolveZkLedgersRootPath(conf);
-        LOG.info("Over Replicated Ledger Deletion : enabled=" + enableGcOverReplicatedLedger + ", interval="
-                + gcOverReplicatedLedgerIntervalMillis);
+        LOG.info("Over Replicated Ledger Deletion : enabled={}, interval={}, maxConcurrentRequest={}",
```

Comment:
Typo: maxConcurrentRequest > maxConcurrentRequests

Reply:
Fixed

Reply:
@eolivelli 
can you please review again and merge this PR?